grails-app/migrations/0.8.x/changelog-2021-03-16-1250-insert-reference-number-type-vendor-invoice-number.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="dhatala" id="1536634735965-2">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                <![CDATA[SELECT count(*) FROM reference_number_type WHERE name="Vendor Invoice Number"]]>
+            </sqlCheck>
+        </preConditions>
+        <insert tableName="reference_number_type">
+            <column name="id" valueNumeric="11" />
```

Comment:
Replace `11` with `VENDOR_INVOICE_NUMBER`

Reply:
@jmiranda I'm getting this error " Caused By: Unknown column 'VENDOR_INVOICE_NUMBER' in 'field list' after changing it